ParseFacebookUtils/Internal/AuthenticationProvider/PFFacebookAuthenticationProvider.h

```diff
@@ -40,8 +36,11 @@ extern NSString *const PFFacebookUserAuthenticationType;
 /// @name Authenticate
 ///--------------------------------------
 
-- (BFTask *)authenticateAsyncWithReadPermissions:(nullable NSArray PF_GENERIC(NSString *) *)readPermissions
-                              publishPermissions:(nullable NSArray PF_GENERIC(NSString *) *)publishPermissions;
+- (BFTask PF_GENERIC(NSDictionary<NSString *,NSString *>*)*)authenticateAsyncWithReadPermissions:(nullable NSArray PF_GENERIC(NSString *)*)readPermissions
```

Comment:
I assume this is auto-format screwing us over, but space between the two `NSString` parameters? If it is just auto format please ignore.